specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json

```diff
@@ -0,0 +1,1938 @@
+          "Copy custom model": {
+            "$ref": "./examples/CopyModelAuthorization.json"
+          }
+        }
+      }
+    },
+    "/custom/models/compose": {
+      "post": {
+        "tags": [
+          "Form"
+        ],
+        "summary": "Compose trained with labels models into one composed model.",
+        "description": "Compose request would include list of models ids.\r\nIt would validate what all models either trained with labels model or composed model.\r\nIt would validate limit of models put together.",
+        "operationId": "ComposeCustomModels",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "name": "composeRequest",
+            "in": "body",
+            "description": "Compose models",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ComposeRequest"
+            }
+          }
+        ],
+        "responses": {
+            "201": {
+              "description": "Request is queued successfully.",
+              "headers": {
+                "Location": {
+                  "type": "string",
+                  "description": "Location and ID of the composed model. The status of composed model is specified in the status property at the model location."
+                }
+              }
+            },
+            "default": {
+              "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            }
+        },
+        "x-ms-examples": {
+            "Compose custom models": {
+              "$ref": "./examples/ComposeModels.json"
+            }
+        }
+      }
+    },
+    "/prebuilt/businessCard/analyze": {
+      "post": {
+        "summary": "Analyze Business Card",
+        "description": "Extract field text and semantic values from a given business card document. The input document must be of one of the supported content types - 'application/pdf', 'image/jpeg', 'image/png' or 'image/tiff'. Alternatively, use 'application/json' type to specify the location (Uri) of the document to be analyzed.",
+        "operationId": "AnalyzeBusinessCardAsync",
+        "consumes": [
+          "application/pdf",
+          "application/json",
+          "image/jpeg",
+          "image/png",
+          "image/tiff"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "includeTextDetails",
+            "in": "query",
+            "description": "Include text lines and element references in the result.",
+            "required": false,
+            "default": false,
+            "type": "boolean",
+            "x-nullable": false
+          },
+          {
+            "name": "locale",
```

Comment:
Would it make sense to include an enum for supported values?  If not, it would be good to return the list of supported values in the error response if an unsupported locale is sent.